scrapers/scrape_ne.py

```diff
@@ -1,123 +1,18 @@
 #!/usr/bin/env python3
```

Comment:
Please add the following line after this:

```
# -*- coding: utf-8 -*-
```

Reply:
Done